DPGAnalysis/HcalTools/plugins/CMTRawAnalyzer.cc

```diff
@@ -9662,7 +9659,7 @@ void CMTRawAnalyzer::fillDigiAmplitudeQIE11(QIE11DataFrame qie11df) {
   double amplitude345 = 0.;
   double ampl = 0.;
   double ampl3ts = 0.;
-  double amplmaxts = 0.;
+  //  double amplmaxts = 0.;
```

Comment:
Please remove the commented out code with these useless variables, here and below

Reply:
OK, these commented lines can be removed - thanks.